.ci/scripts/run_fb_dlrm.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -eEx
```

Comment:
rename .ci/scripts/run_fb_dlrm.sh  to .ci/scripts/run_dlrm.sh

Reply:
Fixed.